packages/graphql-codegen-cli/src/config.ts

```diff
@@ -142,7 +186,16 @@ export async function loadCodegenConfig({
 }
 
 export async function loadContext(configFilePath?: string): Promise<CodegenContext> | never {
-  const graphqlConfig = await findAndLoadGraphQLConfig(configFilePath);
+  let graphqlConfig: GraphQLConfig | undefined | void;
+  try {
+    graphqlConfig = await findAndLoadGraphQLConfig(configFilePath);
+  } catch (err) {
+    if (isRequireESMError(err)) {
+      // TODO: This needs a fix in graphql-config
```

Comment:
I don't think it's a blocker but something that we should fix sooner or later. What do you think?

Reply:
I agree, especially because graphql-config is quite popular.